uefi-test-runner/src/proto/shell.rs

```diff
@@ -1,13 +1,168 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
-use uefi::boot;
+use uefi::boot::ScopedProtocol;
 use uefi::proto::shell::Shell;
+use uefi::{boot, cstr16};
+use uefi_raw::Status;
+
+/// Test `get_env()`, `get_envs()`, and `set_env()`
+pub fn test_env(shell: &ScopedProtocol<Shell>) {
+    /* Test retrieving list of environment variable names */
+    let mut cur_env_vec = shell.get_envs();
+    assert_eq!(cur_env_vec.next().unwrap(), cstr16!("path"),);
+    assert_eq!(cur_env_vec.next().unwrap(), cstr16!("nonesting"),);
```

Comment:
Where does `nonesting` come from?! Looks a little magic to me

Reply:
Interesting, it is pre-defined: https://uefi.org/sites/default/files/resources/UEFI_Shell_2_2.pdf